python/tests/backends/test_oqc.py

```diff
@@ -16,7 +16,11 @@
 import cudaq
 from cudaq import spin
 
-from utils.mock_qpu.oqc import startServer
+try:
+    from utils.mock_qpu.oqc import startServer
+except:
```

Comment:
Why the general except catch here?

Reply:
This change was to hopefully eliminate the errors here: https://github.com/NVIDIA/cuda-quantum/actions/runs/5979874189/job/16225230449?pr=435
That's how the other backends do it.
Basically, those mock server test environments aren't always enabled in every build environment.